templates/default/livy-health.sh.erb

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+set -e
+curl --fail -X GET "http://$HOSTNAME:<%= node['livy']['port'] %>/sessions?from=0&size=1"
```

Comment:
https://superuser.com/a/657174 maybe `--fail` is not that safe?

Reply:
I'm not sure where the `But it blocks output to the screen.` comes from. I've never experienced such issue. I can add the `--silent` flag but I don't see why `--fail` is not safe

Reply:
Let's keep it as it is then

Reply:
Actually I have forgotten to redirect stdout and stderr to /dev/null So I'm adding the `--silent` flag and redirection of outputs ;)